scraper.rb

```diff
@@ -83,11 +151,11 @@ def process(html)
 
     test_meeting = meeting_rows.first
 
-    meeting_rows.each do |meeting_row|
+    meeting_rows.reverse.each do |meeting_row|
       meeting = parse_meeting(meeting_row)
       download_meeting(meeting)
     end
   end
 end
 
-AgendaScraper.new.initialize
+AgendaScraper.new.init
```

Comment:
I'm going to make this more ruby like in a future PR but if you keep it `initialize`, `AgendaScraper.new` will call initialize so we don't have to do `.new.init`

Usually we would do something like `AgendaScraper.new(params).process` or `AgendaScraper.new(params).call`